.claude/commands/code-complexity-check.md

```
@@ -0,0 +1,172 @@
+# Fail CI if critical complexity found
+/code-complexity-check --strict
+
+# JSON output for CI
+/code-complexity-check --json
+```
+
+## Complexity Calculation
+
+Cyclomatic complexity is calculated as:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)